AltSign/Model/Apple API/ALTCertificate.h

```diff
@@ -16,8 +16,8 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, copy) NSString *serialNumber;
 
 @property (nonatomic, copy, nullable) NSString *identifier;
-@property (nonatomic, copy, nullable) NSString *machineName;
-@property (nonatomic, copy, nullable) NSString *machineIdentifier;
+@property (nonatomic, copy, nullable) id machineName;
+@property (nonatomic, copy, nullable) id machineIdentifier;
```

Comment:
Changing these property types from `NSString` (which it *should* be) to `id` doesn't really seem to address the underlying issue, which is that when parsing certificates the machineName and machineIdentifier keys might be "null" (from what I understand). Plus, this makes working with this library from Swift much more frustrating due to type erasure.

Reply:
This API is from Apple, they use `NSNull`. You have to do compatible hack...
If I'm the author of Apple's Private Framework, I will not use suck a hack logic (return NSNull instead of nil)

Reply:
I understand, but it'd be better to update `-[ALTCertificate initWithResponseDictionary:]` to explicitly check for NULL values and assign nil instead to those properties, rather than assign the NSNull object directly and change the public API.